packages/turf/index.js

```diff
@@ -16,7 +16,7 @@ var turf = {
     within: require('@turf/within'),
     concave: require('@turf/concave'),
     difference: require('@turf/difference'),
-    dissolve: require('@turf/dissolve'),
+    // dissolve: require('@turf/dissolve'),
```

Comment:
Damn.. still can't make the tests pass since it really needs that `@turf/dissolve` module to be present for `@turf/turf`, once the package is published once it will be ok.

https://travis-ci.org/Turfjs/turf/builds/194629215#L2380-L2383

You can uncomment this since the tests has passed all the way to the end.